index.md

```diff
@@ -45,7 +47,7 @@ layout: page
   </div>
 </div>
 
-<hr>
+<div><hr></div>
```

Comment:
This seems a little awkward to me.  I haven't looked into the `<p>` tag issue you mentioned.  Is there any other way to avoid it without wrapping `<hr>` in `<div>`?  If not, this solution is probably better than `<p>`.